api/migrations/versions/14d53fa0c9c8_.py

```diff
@@ -0,0 +1,556 @@
+    countries = session.query(Country).all()
+    country_list = []
+    for country in countries:
+        country_list.append({
+            'label': country.name,
+            'value': country.name
+        })
+    return country_list
+
+
+def upgrade():
+    # pass
+    Base.metadata.bind = op.get_bind()
+    session = orm.Session(bind=Base.metadata.bind)
+
+    kambuledoctoral2020 = Event('Thamsanqa Kambule Doctoral Dissertation Award 2020',
+    'Thamsanqa Kambule Doctoral Dissertation Award 2020',
+    datetime.date(2020, 8, 23),datetime.date(2020, 8, 28), 'kambuledoctoral2020',
+    1,'baobab@deeplearningindaba.com','http://www.deeplearningindaba.com',
+    datetime.date(2020,3,1), datetime.date(2020,4,17),datetime.date(2020,4,25),
+    datetime.date(2020,5,15),datetime.date(2020,1,1),datetime.date(2020,1,1),
+    datetime.date(2020,1,1), datetime.date(2020,1,1),datetime.date(2020,1,1),
+    datetime.date(2020,1,1), EventType.AWARD)
+
+    session.add(kambuledoctoral2020)
+    session.commit()
+
+    event_id = kambuledoctoral2020.id
+
+    application_form = ApplicationForm(event_id, True, True)
+    session.add(application_form)
+    session.commit()
+
+    app_form_id = application_form.id
+
+    main_section = Section(app_form_id,'Thamsanqa Kambule Doctoral Dissertation Award 2020',"""
+    This is the official application form for the Thamsanqa Kambule Doctoral Dissertation Award 2020, an award to encourage and recognise excellence in research and writing by Doctoral candidates at African universities, in any area of computational and statistical science. This award will be made at the Deep Learning Indaba at Tunis, Tunisia, in August 2020. 
+
+This application will require:
+- Personal details about the nominee,
+- Details about the dissertation, including a PDF of the dissertation itself, its abstract and core contributions,
+- 1 supporting letter from a person other than the nominator (submitted separately through Baobab)
+
+For eligibility criteria for the Kambule Doctoral Award, please see www.deeplearningindaba.com/awards
+
+For any queries, please email awards@deeplearningindaba.com.
+    """,1)
+
+    session.add(main_section)
+    session.commit()
+
+    q1_nomination_capacity = Question(
+        application_form_id=app_form_id,
+        section_id=main_section.id,
+        headline='Nomination Capacity',
+        placeholder='',
+        order=1,
+        questionType='multi-choice',
+        validation_regex=1,
+        is_required=True,
+        options=[
+            {'label':'Self-nomination', 'value':'self'},
+            {'label':'Nomination on behalf of a candidate','value':'other'}
+        ]
+    )
+    session.add(q1_nomination_capacity)
+    session.commit()
+
+    nominator_information = Section(app_form_id, 'Nomination Information', """
+Details of the person nominating the doctoral candidate. Dependency: Only if 'Nominating Capacity' question has answer 'Nomination on behalf a candidate'
```

Comment:
Can you fill in the dependency fields here and remove the Dependency part of the description :)

Reply:
No problem! Seems I overlooked it, will change it promptly.